docs/cli-usage.md

```diff
@@ -190,5 +190,42 @@ The best way to find out how that works is playing around with different values
 
 !!! note
 
-    Some identifiers do have text transformation enabled by default. 
+    Some identifiers do have text transformation enabled by default.
     Nevertheless make sure to **explicitly set** text transformation styles if you need them! All text transformation may be disabled by default in a future release of the generator.
+
+### Default Identifier Style settings
+(Please note that this list is work in progress)
+#### C++
+
+| Argument | Default  |
+| -------- | -------- |
+| `--ident-cpp-enum` | FOO_BAR |
+| `--ident-cpp-field` | foo_bar |
+| `--ident-cpp-method` | foo_bar |
+| `--ident-cpp-type` | FooBar |
+| `--ident-cpp-enum-type` | FooBar |
+| `--ident-cpp-type-param` | FooBar |
+| `--ident-cpp-local` | foo_bar |
+| `--ident-cpp-file` | foo_bar |
+
+#### Java
+
+| Argument | Default  |
+| -------- | -------- |
+| `--ident-java-enum` | FOO_BAR |
+| `--ident-java-field` | fooBar |
+| `--ident-java-type` | FooBar |
+| `--ident-jni-class` | FooBar |
+| `--ident-jni-file` | foo_bar |
+
+#### Objective C
+
+| Argument | Default      |
+| -------- | -------- |
+| `--ident-objc-enum` | FooBar |
+| `--ident-objc-field` | fooBar |
+| `--ident-objc-method` | fooBar |
+| `--ident-objc-type` | FooBar |
+| `--ident-objc-type-param` | FooBar |
+| `--ident-objc-local` | fooBar |
+| `--ident-objc-file` | FooBar |
```

Comment:
I prefer to put the default-values in `code`-blocks to make clear it's a parameter value.  But I'm not too passionate about it, it's up to you. 

```suggestion
| `--ident-cpp-enum` | `FOO_BAR` |
| `--ident-cpp-field` | `foo_bar` |
| `--ident-cpp-method` | `foo_bar` |
| `--ident-cpp-type` | `FooBar` |
| `--ident-cpp-enum-type` | `FooBar` |
| `--ident-cpp-type-param` | `FooBar` |
| `--ident-cpp-local` | `foo_bar` |
| `--ident-cpp-file` | `foo_bar` |

#### Java

| Argument | Default  |
| -------- | -------- |
| `--ident-java-enum` | `FOO_BAR` |
| `--ident-java-field` | `fooBar` |
| `--ident-java-type` | `FooBar` |
| `--ident-jni-class` | `FooBar` |
| `--ident-jni-file` | `foo_bar` |

#### Objective C

| Argument | Default      |
| -------- | -------- |
| `--ident-objc-enum` | `FooBar` |
| `--ident-objc-field` | `fooBar` |
| `--ident-objc-method` | `fooBar` |
| `--ident-objc-type` | `FooBar` |
| `--ident-objc-type-param` | `FooBar` |
| `--ident-objc-local` | `fooBar` |
| `--ident-objc-file` | `FooBar` |
```